test/instrumentation/run.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/sh
+version=`node -v`
+versionMinusV=`echo $version | cut -c 2-`
+nodeRoot="node-$versionMinusV"
```

Comment:
Do we have a Shell scripting style guide @mattrobenolt @JTCunning?

e.g. something like https://google.github.io/styleguide/shell.xml#Naming_Conventions